includes/classes/Feature/Facets/Types/Taxonomy/FacetType.php

```diff
@@ -172,19 +172,19 @@ public function facet_query( $query ) {
 
 		$tax_query = $query->get( 'tax_query', [] );
 
-		// Account for taxonomies that should be woocommerce attributes, if WC is enabled
-		$attribute_taxonomies = [];
-		if ( function_exists( 'wc_attribute_taxonomy_name' ) ) {
-			$all_attr_taxonomies = wc_get_attribute_taxonomies();
-
-			foreach ( $all_attr_taxonomies as $attr_taxonomy ) {
-				$attribute_taxonomies[ $attr_taxonomy->attribute_name ] = wc_attribute_taxonomy_name( $attr_taxonomy->attribute_name );
-			}
-		}
+		/**
+		 * Filter for treatment special slugs in taxonomies. This is used in case you need to change the default taxonomy slug
+		 *
+		 * @since 4.7.0
+		 * @hook ep_facet_tax_special_slug_taxonomies
+		 * @param  {array} $special_taxonomies Taxonomies with special slugs
+		 * @return {array} $selected_filters Selected filters
+		 */
```

Comment:
We don't need to repeat the full docblock here, it is enough to change this to
```suggestion
		/** This filter is documented below */
```
like WordPress core does (in WP they mention the same file name, which I think is a redundancy.)

Reply:
I've removed the docblock @felipeelia